paasta_tools/list_tron_namespaces.py

```diff
@@ -34,14 +34,24 @@ def parse_args():
         default=tron_tools.DEFAULT_SOA_DIR,
         help="Use a different soa config directory",
     )
+    parser.add_argument(
+        "-e",
+        "--executors",
+        default=["paasta"],
```

Comment:
I am ok either way. Let me know.

Reply:
I think just modifyign th edefault to match status quo (including all executor types) is the easiest route.

You'd have to work on timing both the paasta upgrade + puppet change at the same time on tron servers otherwise, which can be annoying.

Reply:
I think the executors are set as `paasta, spark` which should cover all executor types right?
Why do we need to match the timing? 
Rollout: 
* Rollout this change as `paasta, spark` # work as status-quo
* Puppet: change `executors` to `paasta` only # better solution
* PUppet: add cron job for `executors` for `spark`

Reply:
I changed the default to 'paasta, spark' in my previous commit.
Seems like we are on same page :) 
Resolving this comment unless any other changes are needed.